src/core/providers/QueryParams.tsx

```diff
@@ -1,16 +1,13 @@
 import React, { createContext, ReactNode } from 'react';
-import { RouteComponentProps, withRouter } from 'react-router';
 import { useQueryParamsState } from '../hooks/useQueryParamsState';
 
 export const QueryParams = createContext({} as ReturnType<typeof useQueryParamsState>);
 
-export interface IProps extends RouteComponentProps {
+interface IProps {
   children: ReactNode;
 }
 
-export const QueryParamsProvider = ({ location, children }: IProps) => {
-  const queryParams = useQueryParamsState(location);
+export const QueryParamsProvider = ({ children }: IProps) => {
+  const queryParams = useQueryParamsState();
```

Comment:
This doesn't need an argument anymore? Neat!

Reply:
Managed to go around it, we could have avoided it with React Router as well, but I guess it was never done